src/server/utils/cassandra.js

```diff
@@ -13,11 +14,33 @@ export function getCassandraClientAsync(keyspace) {
   if (clientPromises.has(keyspace)) {
     return clientPromises.get(keyspace);
   }
+
```

Comment:
The keyspace, which was already configurable, is being passed into this function. Rather than doing this block where you manually check environment variables, I would change the function signature to expect keyspace, username, and password where the username and password could be defaults. Then, I would use the `config` module (just like we do for the keyspace) to pass any configured username/password into this function.